src/style.css

```diff
@@ -45,13 +48,38 @@ footer {
   padding: 10px 50px;
   cursor: pointer;
   transition: all 0.5s ease;
+  outline: none;
 }
 .create-task-btn:hover {
   color: white;
   background-color: #1d1283;
   border: none;
 }
 
+/* profile section */
+.profileSection {
+  height: 80vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.profile-task-btn {
+  color: black;
+  font-weight: 500;
+  font-size: larger;
+  background-color: white;
+  border: 2px solid black;
+  border-radius: 5px;
+  padding: 10px 50px;
+  cursor: pointer;
+  transition: all 0.5s ease;
+}
+.profile-task-btn:hover {
+  color: white;
+  background-color: #1d1283;
+  border: none;
+}
+
```

Comment:
At some places, we are using **rems** and at some places, we are using **px**
It's a good practice to have consistency

Reply:
Fixed.